pkg/controller/kibana/driver.go

```diff
@@ -287,6 +251,21 @@ func (d *driver) deploymentParams(kb *kbv1.Kibana) (deployment.Params, error) {
 	}, nil
 }
 
+func (d *driver) buildVolumes(kb *kbv1.Kibana) []commonvolume.VolumeLike {
+	volumes := []commonvolume.VolumeLike{DataVolume, ConfigSharedVolume, ConfigVolume(*kb)}
```

Comment:
Are we mounting `ConfigVolume()` (if I'm not mistaken: the "raw" config from the secret resource) into the main Kibana container? Isn't it only required for the init container?

Reply:
The Secret with the configuration is needed because I eventually decided to create some symbolic links. Even if it is not strictly required for Kibana I think that it is easier to reason about something that is close to what is done with Elasticsearch.